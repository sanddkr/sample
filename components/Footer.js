import React, { Component } from 'react';
import { Link } from 'react-router-dom';
export class Footer extends Component {
  constructor(props) {
    super(props);
    this.state= {};
  }
  render() {
    return (
      <div className="wrapper footerwrap fixed-bottom">
        <div className="container">
        <header className="text-center">Disclaimer</header>
      </div>
    </div>
    )
  }
}