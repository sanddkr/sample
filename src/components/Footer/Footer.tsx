import * as React from 'react';
// import { Link } from 'react-router-dom';

export const Footer = (props) => {
  return(
    <div className="wrapper footerwrap fixed-bottom">
      <div className="container">
        <header className="text-center">Disclaimer</header>
      </div>
    </div>
  )
};