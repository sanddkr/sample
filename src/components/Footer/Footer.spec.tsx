import * as React from "react";
import {mount, shallow} from "enzyme";
import * as renderer from 'react-test-renderer';
import Footer from './';

describe("Footer Component", () => {
    let wrapper, mountwrap;

    beforeEach(() => {
        wrapper = shallow(<Footer />);
        mountwrap = mount(<Footer />);
    });

    it('Footer Component renders correctly', () => {
        expect(wrapper.children().length).not.toBeLessThan(1);
    });

    it('Footer Component renders correctly using snapshots', () => {
        const rendered = renderer.create(
            <Footer />
        );
        expect(rendered.toJSON()).toMatchSnapshot();
    });
});