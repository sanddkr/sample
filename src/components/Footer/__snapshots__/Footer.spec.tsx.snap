// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component Footer Component renders correctly using snapshots 1`] = `
<div
  className="wrapper footerwrap fixed-bottom"
>
  <div
    className="container"
  >
    <header
      className="text-center"
    >
      Disclaimer
    </header>
  </div>
</div>
`;
